Makefile

```
@@ -31,7 +33,7 @@ deps: init
 	docker tag $(DOCKER_IMAGE_NAME) $(DOCKER_IMAGE_NAME_BASE)
 
 %.install:
-	@docker run --rm --env DOCKER_IMAGE --env DOCKER_TAG $(DOCKER_IMAGE_NAME) | bash -s $(DOCKER_TAG)
+	@docker run --rm --env APP_NAME --env DOCKER_IMAGE --env DOCKER_TAG --env INSTALL_PATH $(DOCKER_IMAGE_NAME) | bash -s $(DOCKER_TAG)
```

Shouldn't this use `$()` for consistency with `$(DOCKER_IMAGE_NAME)` and `$(DOCKER_TAG)`

```suggestion
	@docker run --rm --env $(APP_NAME) --env $(DOCKER_IMAGE) --env $(DOCKER_TAG) --env $(INSTALL_PATH) $(DOCKER_IMAGE_NAME) | bash -s $(DOCKER_TAG)
```